tests/integration/clientv3/examples/example_metrics_test.go

```diff
@@ -17,13 +17,14 @@ package clientv3_test
 import (
 	"context"
 	"fmt"
+	"github.com/prometheus/client_golang/prometheus"
```

Comment:
Please move this item to line 28

cc @ivanvc @jmhbnz it seems that the workflow check can't detect this minor issue.

Reply:
ACK. doing that now.